### PR TITLE
Feature/write robot

### PR DESCRIPTION
**Pull request by nimitsarup**
- function to write a robots file
- config file driven
- unit + feature tests

### CODE REVIEW

File: features/filewriter.feature [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+User-agent: GoogleBot
+Allow: /allow1
+Allow: /allow2
+Disallow: /deny1
+Disallow: /deny2
+
+sitemap: www.site1.com/sitemap1
+sitemap: www.site2.com/sitemap2
+
+        """
```
Comment (cookel2) [inline features/filewriter.feature]: There should be a new line at the end of the file.

File: features/steps/steps.go [hunk 2]
```diff
@@ -2,90 +2,42 @@ package steps
 
 import (
 	"context"
-	"io/ioutil"
+	"fmt"
 	"os"
-	"os/signal"
-	"syscall"
-	"time"
+	"strings"
```
Comment (cookel2) [inline features/steps/steps.go:7]: The imports need to be organised e.g. delete line 10, which is blank.

File: robotseo/robot_test.go [hunk 3]
```diff
@@ -79,11 +80,12 @@ func TestInit(t *testing.T) {
 
 func TestGetRobotsFileBody(t *testing.T) {
 	var expectedRobotsBody string
+	r := RobotFileWriter{}
 
 	Convey("no robots data", t, func() {
 		robotList = map[string]asset.SeoRobotModel{}
 		expectedRobotsBody = ""
-		So(GetRobotsFileBody(), ShouldEqual, expectedRobotsBody)
+		So(r.GetRobotsFileBody(), ShouldEqual, expectedRobotsBody)
```
Comment (greg-fujar) [inline robotseo/robot_test.go:87]: should these tests be moved to file_writer_test.go?

File: config/config.go [hunk 4]
```diff
@@ -74,7 +74,7 @@ func Get() (*Config, error) {
 		HealthCheckCriticalTimeout: 90 * time.Second,
 		SitemapGenerationFrequency: time.Hour,
 		SitemapGenerationTimeout:   10 * time.Minute,
-		OutputFilePath:             "/tmp/helloworld.txt",
+		RobotsFilePath:             "./dp_robot_file.txt",
```
Comment (greg-fujar) [inline config/config.go:77]: maybe add to config test?

**Review by cookel2 — COMMENTED** [review]
Just a couple of comments. Also it would be good to format the project (run ```go fmt ./...``` from the project root.

**Review by greg-fujar — APPROVED** [review]

**Review by cookel2 — APPROVED** [review]
LGTM

**Pull request merged**